eng/validation/build-arcadewithrepo.ps1

```diff
@@ -250,27 +318,30 @@ while("completed" -ne (Get-BuildStatus -buildId $buildId))
     Start-Sleep -Seconds (5*60)
 }
 
-## Output summary of references for investigations
-Write-Host "Arcade Version: ${arcadeSdkVersion}"
-Write-Host "Repository Cloned: ${githubOrg}/${targetRepoName}"
-Write-Host "Branch name in repository: ${targetBranch}"
-Write-Host "Last Known Good build SHA: ${sha}"
-
 ## If build fails, then exit
 $buildResult = (Get-BuildResult -buildId $buildId)
 if(("failed" -eq $buildResult) -or ("canceled" -eq $buildResult))
 {
-    Write-Error "Build failed"
+    Write-Error "Build failed or was cancelled"
     exit
 }
 
 ## Clean up branch if successful
+Write-Host "Build was successful. Cleaning up ${targetBranch} branch."
 try
 {
-    & darc delete-default-channel --channel "General Testing" --branch $darcBranchName --repo $darcRepoName --github-pat $githubPAT --password $barToken
-    Git-Command $targetRepoName push origin --delete $targetBranch
+    if($true -eq $pushBranchToGithub)
+    {
+        & darc delete-default-channel --channel "General Testing" --branch $darcBranchName --repo $darcGitHubRepoName --github-pat $githubPAT --password $barToken
+        Git-Command $githubRepoName push origin --delete $targetBranch
+    }
+    else
+    {
+        & darc delete-default-channel --channel "General Testing" --branch $darcBranchName --repo $darcAzDORepoName --azdev-pat $azdoToken --password $barToken
+        Git-Command $githubRepoName push $remoteName --delete $targetBranch
+    }
 }
 catch
 {
-    Write-Warning "Unable to delete default or branch when cleaning up branch"
+    Write-Warning "Unable to delete default channel or branch when cleaning up branch"
```

Comment:
```suggestion
    Write-Warning "Unable to delete default channel or branch when cleaning up"
```

Reply:
Sorry, no co-author for you :(  I needed to change it in another place, too.